app/src/features/surveys/components/BlockForm.tsx

```diff
@@ -0,0 +1,38 @@
+import Typography from '@mui/material/Typography';
+import { Box } from '@mui/system';
+import CustomTextField from 'components/fields/CustomTextField';
+import React from 'react';
+
+export interface IBlockData {
+  survey_block_id: number | null;
+  name: string;
+  description: string;
+}
+
+const BlockForm: React.FC = () => {
+  return (
+    <form>
+      <Box>
+        <Box component={'fieldset'} mb={4}>
+          <Typography component="legend">Name and Description</Typography>
+          <Box mt={0.5} mb={3}>
+            <CustomTextField
+              name="name"
+              label="Name"
+              maxLength={50}
+              other={{ placeholder: 'Maximum 50 characters', required: true }}
+            />
+          </Box>
+          <CustomTextField
+            name="description"
+            label="Description"
+            maxLength={250}
+            other={{ multiline: true, placeholder: 'Maximum 250 characters', required: true, rows: 3 }}
```

Comment:
Ideally this should be enforced using Yup schema. See: https://github.com/bcgov/biohubbc/pull/1083/files#diff-28a6bb0e6fc7b03b7de1b8d12c0f779a870fc541953a0352dd61f59ea07bf772

Reply:
I'll update it for both. I did have a conversation with Jer about a similar situation for funding sources. It's a strange interaction if the database throws an error about max length but you still let the user input that much. Instead of validating it just to restrict the input. I'll definitely add that to the yup schema